### PR TITLE
Add darkroomjs@2.0.1 w/ git auto-update

### PR DESCRIPTION
**Pull request by natetang**
Pull request for issue: #11187 
Related issue(s): # #

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/MattKetmo/darkroomjs.git
 * Official website (optional, not the repository): https://mattketmo.github.io/darkroomjs/
 * NPM package url (optional): https://www.npmjs.com/package/darkroom
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 39
   - Count of stars: 864
   - Count of forks: 281
 * NPM download stats (optional): 
   - Downloads in the last day: 1
   - Downloads in the last week: 6
   - Downloads in the last month: 31

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [ ] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/darkroomjs/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  },
+  "author": "Matthieu Moquet <matthieu@moquet.net> (https://moquet.net/)",
+  "autoupdate": {
+    "source": "git",
+    "target": "https://github.com/MattKetmo/darkroomjs.git",
+    "fileMap": [
+      {
+        "basePath": "build",
+        "files": [
+          "darkroom*"
```
Comment (pvnr0082t) [inline ajax/libs/darkroomjs/package.json]: @natetang Let's use `**/*` here.

File: ajax/libs/darkroomjs/package.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  "license": "MIT",
+  "homepage": "https://mattketmo.github.io/darkroomjs",
+  "keywords": [
+    "image",
+    "canvas",
+    "crop"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/mattketmo/darkroomjs.git"
```
Comment (maruilian11) [inline ajax/libs/darkroomjs/package.json]: Let's use `https://github.com/Mattketmo/darkroomjs.git` to keep the consistency, thanks!

**Comment by maruilian11** [comment]
For cdnjs members: may take times to check if it is need to add versions by tracking log.

**Comment by sufuf3** [comment]
Help rebase with the latest master.

**Comment by PeterDaveHello** [comment]
@natetang 👍

**Pull request merged**